ResidentsSocialCarePlatformApi/V1/Boundary/Requests/GetRelationshipsRequest.cs

```diff
@@ -0,0 +1,22 @@
+using System.Text.Json.Serialization;
+using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ResidentsSocialCarePlatformApi.V1.Boundary.Requests
+{
+    public class GetRelationshipsRequest
+    {
+        [FromRoute]
+        public long PersonId { get; set; }
+    }
+
+    public class GetRelationshipsRequestValidator : AbstractValidator<GetRelationshipsRequest>
```

Comment:
I believe we also need to tell `Startup.cs` about our validator.

https://docs.fluentvalidation.net/en/latest/aspnet.html#automatic-registration